composer.json

```diff
@@ -16,6 +16,7 @@
     "require": {
         "php": "^7.4|^8.0",
         "antidot-fw/framework": "^0.1.3",
+        "drift/server": "@dev",
```

Comment:
Is this proper to install unstable dependency for the framework?

Reply:
Thanks @peter279k, Is still working in progress, Driftphp already does not support PHP8, and the release of drift that allows integrating by using an adapter is not released yet. it must be a stable versión before release.

Reply:
@peter279k maybe it will be better to extract this adapter to a third package, allowing to iron the application both Drift and our built-in server

Reply:
Just notice that the `^0.1.17` version is supported for the `php-8.0`.

And the reference is available [here](https://github.com/driftphp/server/blob/0.1.17/.circleci/config.yml).

Reply:
You can work with this new version - 0.1.18 - https://github.com/driftphp/server/releases/tag/0.1.18

This server has almost changed it's public API and in case it changes the public API (interface), it will do it in BC mode.